bundle/libraries/local_path.go

```diff
@@ -77,6 +77,11 @@ func IsLibraryLocal(dep string) bool {
 var packageRegex = regexp.MustCompile(`^[a-zA-Z0-9\-_]+\s?(\[.*\])?\s?((==|!=|<=|>=|~=|==|>|<)\s?\d+(\.\d+){0,2}(\.\*)?)?$`)
 
 func isPackage(name string) bool {
+	// If the name contains comma, it's a package name with miultiple version specifiers
+	if strings.Contains(name, ",") {
+		return true
+	}
```

Comment:
Can you incorporate this into the regexp instead? It could help to initialize the regexp from an `init()` function, then you could reuse the version spec.